tests/testthat/test-verb-expand.R

```diff
@@ -0,0 +1,122 @@
+test_that("expand completes all values", {
+  df <- memdb_frame(x = 1:2, y = 1:2)
+  out <- expand(df, x, y) %>% collect()
+  expect_equal(nrow(out), 4)
+})
+
+test_that("multiple variables in one arg doesn't expand", {
```

Comment:
Can you remove all the tidyr specific tests and start fresh with tests of the basic behaviour. I _think_ they should probably mostly snapshot the generated sql, with a few actuallyy executing just to check that everything is plumbed together correctly.